source/core/parameters.cc

```diff
@@ -958,6 +958,9 @@ namespace Parameters
       Patterns::Double(),
       "position relative to the center of the particle  for the location of the point where the pressure is impose inside the particle  in z ");
     prm.declare_entry("radius", "0.2", Patterns::Double(), "Particles radius ");
+    prm.declare_entry("density", "1", Patterns::Double(), "density of the particle ");
+    prm.declare_entry("inertia", "1", Patterns::Double(), "uniform rotational inertia");
```

Comment:
Why uniform rotation inertia? Should it not be moment of inertia? Or you want to say uniform because here it is not a tensor?

Reply:
Even though it's defined as a tensor in the solver, since the parameter only take a single value the moment of inertia is uniform.